src/pages/Offer/sections/InsuranceCoverage.tsx

```diff
@@ -335,57 +344,116 @@ export const InsuranceCoverage: React.SFC = () => (
               {(text) => text}
             </TranslationsConsumer>
           </PerilInfo>
-          {PERILS.map((peril) => (
-            <BigCol key={peril.key}>
-              {state.activeTab === peril.key ? (
-                <Row>
-                  {peril.icons.map((column) => (
-                    <Col
-                      key={column.key}
-                      onClick={() =>
-                        state.showIconNumber === column.key &&
-                        state.showPerilNumber === peril.key
-                          ? state.handleSameIconClick()
-                          : state.handleIconClick(
-                              peril.key,
-                              column.key,
-                              column.expandableText,
-                            )
-                      }
-                    >
-                      <PerilIcon
-                        src={
-                          state.showIconNumber !== undefined
-                            ? state.showPerilNumber === peril.key &&
-                              state.showIconNumber === column.key
-                              ? column.icon
-                              : column.iconGrey
-                            : column.icon
-                        }
-                      />
-                      <PerilTitle
-                        style={{
-                          color:
-                            state.showIconNumber !== undefined
-                              ? state.showPerilNumber === peril.key &&
-                                state.showIconNumber === column.key
-                                ? colors.DARK_PURPLE
-                                : colors.DARK_GRAY
-                              : colors.DARK_PURPLE,
-                        }}
-                      >
-                        {column.title}
-                      </PerilTitle>
-                    </Col>
-                  ))}
-                </Row>
-              ) : null}
-            </BigCol>
-          ))}
-          {state.showPerilNumber !== undefined &&
-          state.showIconNumber !== undefined ? (
-            <DropDownText>{state.textToShow}</DropDownText>
-          ) : null}
+
+          {PERILS.map(
+            (peril) =>
```

Comment:
coming back to this i realise this is _quite_ heavy. Can we maybe make the peril into a separate component?

Reply:
yes maybe it is, but I don't think extracting everything but the header really changes much:/

Reply:
I reverted to the old design, so nothing new here now